compare.py

```diff
@@ -0,0 +1,80 @@
+from openvino.inference_engine import IECore
+import numpy as np
+import torch
+import torch.onnx
+import openvino.inference_engine as ie
+print(ie.__version__)
+
+from direct.data.tests.test_transforms import *
+from omegaconf import OmegaConf
+from direct.environment import *
+
+
+input_image = np.load('ref/input_image.npy')
+sensitivity_map = np.load('ref/sensitivity_map.npy')
+masked_kspace = np.load('ref/masked_kspace.npy')
+sampling_mask = np.load('ref/sampling_mask.npy')
+loglikelihood_scaling = np.load('ref/loglikelihood_scaling.npy')
+previous_state = np.load('ref/previous_state.npy')
+
+inputs = {}
+inputs['input_image'] = input_image
+inputs['masked_kspace'] = masked_kspace
+inputs['sampling_mask'] = sampling_mask
+inputs['sensitivity_map'] = sensitivity_map
+inputs['loglikelihood_scaling'] = loglikelihood_scaling
+
+
+ie = IECore()
+ie.add_extension('/home/alikholat/projects/openvino_pytorch_layers/user_ie_extensions/build/libuser_cpu_extension.so', 'CPU')
```

Comment:
Please test https://pypi.org/project/openvino-extensions.

Reply:
How can I install [`openvino-extensions`](https://pypi.org/project/openvino-extensions) for openvino 2021.3?

Reply:
It's just runtime so you can convert a model by MO 2021.3 but run with 2021.4. To make so, create a virtual environment:

```bash
python3 -m venv venv3
source venv3/bin/activate
pip install --upgrade pip
pip install git+git://github.com/openvinotoolkit/openvino.git@releases/2021/3#subdirectory=model-optimizer
```